model/src/application/interaction/application_command/data/resolved.rs

```diff
@@ -37,8 +37,16 @@ pub struct InteractionChannel {
 
 #[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
 pub struct InteractionMember {
+    /// Member's guild avatar.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub avatar: Option<String>,
     pub joined_at: Timestamp,
     pub nick: Option<String>,
+    /// Whether the user has yet to pass the guild's Membership Screening
+    /// requirements.
+    pub pending: bool,
+    /// Total permissions of the member in this channel including overwrites
+    pub permissions: Permissions,
```

Comment:
Are these fields guaranteed never null?

Reply:
According to the docs and my tests they are present all the time.